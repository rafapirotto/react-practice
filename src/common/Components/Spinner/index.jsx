import React from 'react';

import spinner from '../../assets/spinner.gif';

const Spinner = () => (
  <div className="mt-40 flex justify-center">
    <img src={spinner} alt="spinner" />
  </div>
);

export default Spinner;
