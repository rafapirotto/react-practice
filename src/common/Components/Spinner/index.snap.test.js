import React from 'react';
import renderer from 'react-test-renderer';

import Spinner from '.';

it('matches the snapshot', () => {
  const component = renderer.create(<Spinner />);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
