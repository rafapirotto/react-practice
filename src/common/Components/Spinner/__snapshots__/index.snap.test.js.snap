// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Matches the snapshot 1`] = `
<div
  className="mt-40 flex justify-center"
>
  <img
    alt="spinner"
    src="spinner.gif"
  />
</div>
`;
